docs/CONFIGURATION.md

```diff
@@ -14,15 +48,15 @@ Mountpoint CSI driver. For more details on creating a policy and an IAM role, re
 ["Creating an IAM role"](https://docs.aws.amazon.com/eks/latest/userguide/s3-csi.html#s3-create-iam-role) from the
 EKS User Guide.
 
-The Mountpoint CSI Driver can be configured to ingest credentials via two approaches: globally for the entire 
+The Mountpoint CSI Driver can be configured to ingest credentials via two approaches: globally for the entire
```

Comment:
Nit: Can these single character changes go in their own PR so it doesn't confuse the commit history?

Reply:
I think that should be fine as this is a doc-only change. For long-term, we should probably look for enforcing a standard format for Go (`gofmt`), YAML and Markdown (`Prettier`).

Reply:
I agree with Simon, if we do find any changes like this then let's just make two commits and merge them in separately.

Agree enforcing format checkers would be better, but I'd prefer we remove the noise from PRs and commit history regardless.